### DISCUSSION TITLE
Edge prettier plugin

### DISCUSSION THREAD

**Discussion by realprabs** [adonisjs/core#3989] [Ideas]
I think this would be a great addition to the ecosystem. Is anyone already working on it?

**Comment by Julien-R44** [comment]
I started working on it a few weeks ago. The idea would be to offer a formatter as a CLI tool, as well as a prettier plugin and also as a formatter in the vscode extension.

No ETA for the moment, but I hope to release the first version in early 2023 ( Maybe February ? ). Since I'll have a long vacation at that time I should have a lot of time to work on it

**Reply by grikomsn** [reply]
Recently started working on Adonis and would love a proper formatting solution! How can I help?

**Reply by Julien-R44** [reply]
My vacation was finally cancelled so I was never able to continue working on it, unfortunately 😣
So feel free to take over if you feel like it, it would be awesome to have

**Comment by sajansharmanz** [comment]
Seen this question popping up in discord often lately, so just going to leave this comment here for anyone that comes here in the future.

We've started working on an EdgeJS prettier plugin which can be seen/used here [https://www.npmjs.com/package/prettier-edgejs](https://www.npmjs.com/package/prettier-edgejs)
